### DISCUSSION TITLE
Extend LandScape

### DISCUSSION THREAD

**Discussion by joe-kiang** [structurizr/onpremises#103] [Q&A]
Hey Guys, 

I am running an on-premise installation of Structrizr, I'm trying to create a landscape diagram of systems in the company, and I've tried to extend the Landscape DSL workspace, but I cant seem to find the correct URL for the .dsl.

**Comment by simonbrowndotje** [comment]
I've added a DSL endpoint to the latest build (2024.02.22) ... you can find the link on the shareable workspace summary pages (i.e. the pages where the URL starts `/share` rather than `/workspace`).

<img width="251" alt="Screenshot 2024-02-22 at 12 08 13" src="https://github.com/structurizr/onpremises/assets/1009874/819c0eb8-172f-48ba-92a0-4dc610c3c702">

That said, I'd generally recommend having the landscape DSL file in source code control rather than hosted via an on-premises installation.
